build.gradle

```diff
@@ -423,6 +423,17 @@ task integrationTest(type: Test) {
 	environment(run.environment)
 }
 
+// Create the task that runs the migration script for empty HTML biography fields of persons
+task dbClearBiographies(dependsOn: 'compileJava', type: JavaExec) {
```

Comment:
Isn't this better invoked as a `customChange` through liquibase?

Reply:
I can have a look at that next week.

Reply:
Liquibase's options are too limited (we need access to a lot of the classes).